DevEdu.API/DevEdu.DAL/Repositories/TaskRepository.cs

```diff
@@ -42,37 +42,37 @@ public List<TaskDto> GetTasks()
             return tasks;
         }
 
-        public int AddTask(TaskDto task)
+        public int AddTask(TaskDto taskDto)

```

Comment:
please add list of tags, list of courseDto and list of taskStudentDto to TaskDto

(couldn't add a comment to TaskDto class)